### ISSUE TITLE
TaniwhaTextField-Wish could add into the list

### ISSUE THREAD

**Issue by iceman201** [vsouza/awesome-ios#1786]
TaniwhaTextField
https://github.com/iceman201/TaniwhaTextField

Description: TaniwhaTextField is a lightweight and beautiful swift textfield framework. It got customise floating placeholder display. And also you can highly customise it.

**Comment by victorgama** [comment]
Hi @iceman201! Before adding `TaniwhaTextField` to the list, would you mind improving its readme? Perhaps describing what it does?

After that, please feel free to open a pull request! 🙌

**Comment by iceman201** [comment]
just updated description

**Comment by victorgama** [comment]
Hey @iceman201!
I think you misunderstood me; see, in order to add your project to Awesome iOS, it must have a descriptive readme, explaining what it does, and how people can use it.

The projects below may give you a better idea:
[alexruperez/ARAutocompleteTextView](https://github.com/alexruperez/ARAutocompleteTextView)
[filipstefansson/AutocompleteField](https://github.com/filipstefansson/AutocompleteField)
[viniciusmo/VMaskTextField](https://github.com/viniciusmo/VMaskTextField)
[richa008/CurrencyTextField](https://github.com/richa008/CurrencyTextField)

**Comment by iceman201** [comment]
hi @victorgama I made some changes on readme is that good enough? sorry this is my first framework project

**Comment by victorgama** [comment]
Hi @iceman201! It is better now! And don't worry, we all are learning here! 📚 🤓

Feel free to open a new Pull Request to add your project to the list. You can place it under the [TextField & TextView](https://github.com/vsouza/awesome-ios#textfield--textview) category. Also please remember to respect our [Contribution Guidelines](https://github.com/vsouza/awesome-ios/blob/master/.github/CONTRIBUTING.md).

We can close this issue once your pull request is merged.

Thank you!

**Comment by iceman201** [comment]
https://github.com/vsouza/awesome-ios/pull/1809

**Comment by iceman201** [comment]
it's been merged please close this issue
